.travis.yml

```diff
@@ -17,9 +17,9 @@ env:
 
 matrix:
   include:
-    - php: 5.4
+    - php: 5.5
```

Comment:
We've made the decision to only support 5.5+ in 4.0 this week. But you probably shouldn't sneak this in this unrelated commit, makes it hard to trace back afterwards.